### PR TITLE
Add config gdpr.amp_exception deprecation warning

### PR DESCRIPTION
**Pull request by bsardo**

### CODE REVIEW

**Review by SyntaxNode — APPROVED** [review]

File: config/config.go [hunk 1]
```diff
@@ -214,6 +214,9 @@ func (cfg *GDPR) validate(errs []error) []error {
 	if cfg.HostVendorID < 0 || cfg.HostVendorID > 0xffff {
 		errs = append(errs, fmt.Errorf("gdpr.host_vendor_id must be in the range [0, %d]. Got %d", 0xffff, cfg.HostVendorID))
 	}
+	if cfg.AMPException == true {
+		glog.Warning("gdpr.amp_exception is deprecated and will be removed in a future version. If you need to disable GDPR for AMP, you may do so on a per-account basis or disable it for all request types at the host level.")
```
Comment (mansinahar) [inline config/config.go:218]: Can we please include where they can set this as you did in the deprecation comment above?

**Review by mansinahar — COMMENTED** [review]
LGTM other than the minor comment below

File: config/config.go (hunk 1 above)
Comment (SyntaxNode) [reply config/config.go:218]: How about:
`glog.Warning("gdpr.amp_exception is deprecated and will be removed in a future version. If you need to disable GDPR for AMP, you may do so per-account (gdpr.integration_enabled.amp) or at the host level for the default account (account_defaults.gdpr.integration_enabled.amp).")`

Comment (mansinahar) [reply config/config.go:218]: Yup! That looks great 👍

**Review by SyntaxNode — APPROVED** [review]

**Review by mansinahar — APPROVED** [review]

**Pull request merged**